### PR TITLE
#8235 feature request: zpool iostat N should repeat header like vmstat N

### PR DESCRIPTION
**Pull request by madwizard**
### Motivation and Context

Fulfills #8235 
### Description

zpool_do_iostat:
 - added new int variable that holds default number of rows (24) (winheight),
 - find out the actual number of rows terminal has using ioctl and winsize,
 - every time number of printed rows (tracked by cb.cb_iteration) % winheight == 0, print the header
 - unless -n flag was specified, which reverts to original, one time print of headers

### How Has This Been Tested?

run zpool iostat 1 in terminal of various heights,
run zpool iostat -n 1 in terminal of various heights,
run zpool iostat -H 1 in terminal of various heights,
run zpool iostat -H -n 1 in terminal f various heights,
run zpool iostat -H -n
run zpool iostat -n
run zpool iostat -H
Ran the same tests with numbers different than 1.
 

Ubuntu 18.10, kernel 

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Performance enhancement (non-breaking change which improves efficiency)
- [ ] Code cleanup (non-breaking change which makes code smaller or more readable)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation (a change to man pages or other documentation)

### Checklist:

- [ ] My code follows the ZFS on Linux [code style requirements](https://github.com/zfsonlinux/zfs/blob/master/.github/CONTRIBUTING.md#coding-conventions).
- [x] I have updated the documentation accordingly.
- [x] I have read the [**contributing** document](https://github.com/zfsonlinux/zfs/blob/master/.github/CONTRIBUTING.md).
- [ ] I have added [tests](https://github.com/zfsonlinux/zfs/tree/master/tests) to cover my changes.
- [x] All new and existing tests passed.
- [ ] All commit messages are properly formatted and contain [`Signed-off-by`](https://github.com/zfsonlinux/zfs/blob/master/.github/CONTRIBUTING.md#signed-off-by).

### CODE REVIEW

File: cmd/zpool/zpool_main.c [hunk 1]
```diff
@@ -4771,6 +4774,7 @@ zpool_do_iostat(int argc, char **argv)
 	boolean_t guid = B_FALSE;
 	boolean_t follow_links = B_FALSE;
 	boolean_t full_name = B_FALSE;
+    boolean_t headers_once = B_FALSE;
```
Comment (ghfields) [inline cmd/zpool/zpool_main.c]: nit: Text Alignment (Space/tab)

File: cmd/zpool/zpool_main.c [hunk 2]
```diff
@@ -4844,6 +4848,9 @@ zpool_do_iostat(int argc, char **argv)
 		case 'y':
 			omit_since_boot = B_TRUE;
 			break;
+        case 'n':
```
Comment (ghfields) [inline cmd/zpool/zpool_main.c]: nit: Text Alignment (Space/tab)

File: cmd/zpool/zpool_main.c [hunk 3]
```diff
@@ -5005,6 +5012,10 @@ zpool_do_iostat(int argc, char **argv)
 		return (1);
 	}
 
+
+    if(ioctl(1,TIOCGWINSZ, &win) != -1 && win.ws_row > 0)
```
Comment (ghfields) [inline cmd/zpool/zpool_main.c]: nit: Text Alignment (Space/tab)

File: cmd/zpool/zpool_main.c [hunk 4]
```diff
@@ -5051,9 +5062,10 @@ zpool_do_iostat(int argc, char **argv)
 			 *
 			 * The histogram code explicitly prints its header on
 			 * every vdev, so skip this for histograms.
+             *
```
Comment (ghfields) [inline cmd/zpool/zpool_main.c]: nit: Text Alignment (Space/tab)

**Comment by madwizard** [comment]
I need to figure out git workflow. Closing until I make it right.

**Comment by bunder2015** [comment]
We have a wiki article you might be interested in... https://github.com/zfsonlinux/zfs/wiki/Git-and-GitHub-for-beginners

**Pull request closed without merging**